From 79928996c2bf6ebb143564faf371433489ae5da4 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 5 Sep 2014 12:30:35 +0200
Subject: [PATCH 01/12] spice: move qemu_spice_display_*() from spice-graphics to spice-core

Message-id: <1409920238-9070-2-git-send-email-kraxel@redhat.com>
Patchwork-id: 60875
O-Subject: [RHEL-7.1 qemu-kvm PATCH 1/4] spice: move qemu_spice_display_*() from spice-graphics to spice-core
Bugzilla: 1054077
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 7cc6a25fe94b430cb5a041bcb19d7d854b4e99a7)
---
 ui/spice-core.c    | 16 ++++++++++++++++
 ui/spice-display.c | 17 -----------------
 2 files changed, 16 insertions(+), 17 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 ui/spice-core.c    |   16 ++++++++++++++++
 ui/spice-display.c |   17 -----------------
 2 files changed, 16 insertions(+), 17 deletions(-)

diff --git a/ui/spice-core.c b/ui/spice-core.c
index d67449e..6a7c86f 100644
--- a/ui/spice-core.c
+++ b/ui/spice-core.c
@@ -47,6 +47,7 @@ static const char *auth = "spice";
 static char *auth_passwd;
 static time_t auth_expires = TIME_MAX;
 static int spice_migration_completed;
+static int spice_display_is_running;
 int using_spice = 0;
 int spice_displays;
 
@@ -873,6 +874,21 @@ int qemu_spice_display_add_client(int csock, int skipauth, int tls)
     }
 }
 
+void qemu_spice_display_start(void)
+{
+    spice_display_is_running = true;
+}
+
+void qemu_spice_display_stop(void)
+{
+    spice_display_is_running = false;
+}
+
+int qemu_spice_display_is_running(SimpleSpiceDisplay *ssd)
+{
+    return spice_display_is_running;
+}
+
 static void spice_register_config(void)
 {
     qemu_add_opts(&qemu_spice_opts);
diff --git a/ui/spice-display.c b/ui/spice-display.c
index 82d8b9f..f0f95c8 100644
--- a/ui/spice-display.c
+++ b/ui/spice-display.c
@@ -126,23 +126,6 @@ void qemu_spice_wakeup(SimpleSpiceDisplay *ssd)
     ssd->worker->wakeup(ssd->worker);
 }
 
-static int spice_display_is_running;
-
-void qemu_spice_display_start(void)
-{
-    spice_display_is_running = true;
-}
-
-void qemu_spice_display_stop(void)
-{
-    spice_display_is_running = false;
-}
-
-int qemu_spice_display_is_running(SimpleSpiceDisplay *ssd)
-{
-    return spice_display_is_running;
-}
-
 static void qemu_spice_create_one_update(SimpleSpiceDisplay *ssd,
                                          QXLRect *rect)
 {
-- 
1.7.1

